src/App.js

```diff
@@ -1,23 +1,72 @@
-import logo from './logo.svg';
+import { useState } from 'react';
+import contactArr from './contacts.json'
 import './App.css';
 
+
+
 function App() {
+  let fiveContacts = contactArr.slice(0, 5)
+
+  const [contacts, setContacts] = useState(fiveContacts);
+
+  let otherContacts = contactArr.slice(5, contactArr.length)
```

Comment:
no need to slice the rest contactsArr before filter, since contacts in your state only holds the ones that are displaying already